modules/govuk/manifests/node/s_postgresql_primary.pp

```diff
@@ -21,6 +21,8 @@
     slave_password => $standby_password,
   }
 
+  include govuk::pgbouncer
```

Comment:
Think this needs to be `::govuk::pgbouncer`

Reply:
`s_postgresql_base` doesn't have the leading `::`: https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/node/s_postgresql_base.pp